tools/gen-intrinsics-specs.py

```diff
@@ -41,42 +41,67 @@ def quote_literal(val):
 __SECTION_KEYWORDS = [__INTRINSIC_TABLE_KEYWORD, __SECTION_TEXT_KEYWORD]
 
 
-def rst_literal_quote(mapping):
+def literal_quote(mapping, table_format):
     r"""
-    >>> rst_literal_quote('a; b')
+    >>> literal_quote('a; b', 'grid')
     ' ::\n\n    a \n    b \n\n'
 
-    >>> rst_literal_quote('a b')
+    >>> literal_quote('a b', 'grid')
     ' ::\n\n    a b \n\n'
 
-    >>> rst_literal_quote('')
+    >>> literal_quote('', 'grid')
+    ''
+    >>> literal_quote('a; b', 'github')
+    '<br><br>    a <br>    b <br><br>'
```

Comment:
Too many `<br>`s here :)

I think we want to reduce this to:

```
a<br>b
```
We also want to make sure that the `a` and `b` are rendered as fixed size font, so we have to wrap them with `s. The final output I'd expect to see here would be as follows:

```
`a`<br>`b`
```